sw/device/lib/dif/dif_aes.h

```diff
@@ -101,26 +101,71 @@ typedef enum dif_aes_operation {
 typedef enum dif_aes_mode {
   /**
    * The Electronic Codebook Mode.
+   * In ECB cipher mode the key must be changed for every new block of data.
+   * This is the only secure way to use ECB cipher mode.
+   *
+   * Note: ECB cipher mode don't use the iv parameter of the `dif_aes_start`
```

Comment:
Nit: "The ECB cipher mode doesn't use..."

Reply:
Done